go/service/tlf.go

```diff
@@ -58,3 +61,45 @@ func (h *tlfHandler) CompleteAndCanonicalizePrivateTlfName(ctx context.Context,
 	ctx = chat.Context(ctx, arg.IdentifyBehavior, &breaks, chat.NewIdentifyNotifier(h.G()))
 	return h.tlfInfoSource.CompleteAndCanonicalizePrivateTlfName(ctx, arg.TlfName)
 }
+
+func (h *tlfHandler) identifyUser(ctx context.Context, assertion string, private bool, idBehavior keybase1.TLFIdentifyBehavior) (keybase1.TLFIdentifyFailure, error) {
```

Comment:
See comment above.  If there is no identifier but there is a delegated identify ui, this is used.